GeeksCoreLibrary/Core/Extensions/ConfigurationServiceCollectionExtensions.cs

```diff
@@ -50,6 +51,7 @@
 using GeeksCoreLibrary.Modules.HealthChecks.Services;
```

Comment:
Missed this one last time, but line 12 and line 51 are the same, please remove this line as its also has wrong whitespace, line 12 should be enough